config/jobs/kubernetes-sigs/sig-windows/release-1.24-windows.yaml

```diff
@@ -30,6 +30,7 @@ periodics:
   extra_refs:
   - org: kubernetes-sigs
     repo: cluster-api-provider-azure
+    # Stay on release-1.8 branch if kubekins-e2e is earlier than 1.27.
```

Comment:
does k/k have a plan to backport go 1.20 to previous releases as they started doing for the past go version? if so this is only true until kubekins-e2e images get 1.20

Reply:
No idea. I looked around in test-infra but didn't find any clues, but I can try to find out.

Should I close this PR and open separate ones per-project? I'm concerned that trying to update projects other than cluster-api-provider-azure is delaying testing with the new release branch.

Reply:
My read on that backporting was that it was a one-time thing, and not an ongoing thing.

Reply:
I think the only sustainable way forward is to not use kubekins for these CAPZ jobs

Reply:
What if I just changed the comment to:

`# Don't update from release-1.8 without Go 1.20 support in the container image`

Reply:
It was actually an enhancement: https://github.com/kubernetes/enhancements/issues/3744

And looks like backports are coming soon for go-1.20: https://github.com/kubernetes/release/issues/2815#issuecomment-1526400628

In this case @mboersma let's merge this for now and bump the jobs once 1.20 is part of the older kubekins images. Would you be able to please clarify the comments that bumping is TODO dependent on go 1.20 being backported to the test images?

Reply:
Thanks for tracking that down @CecileRobertMichon, happy to learn k/k is going to solve this for us this time :)